src/worker.cc

```diff
@@ -959,6 +961,18 @@ void from_json(rapidjson::Document& doc, Options& options) {
     options.set_resample_distance(*resample_distance);
   }
 
+  // expansion action, default to isochrone
+  auto exp_action_str = rapidjson::get_optional<std::string>(doc, "/action");
+  Options::Action exp_action = Options::isochrone;
+  if (exp_action_str) {
+    static const std::vector<Options::Action> valid_types{Options::route, Options::isochrone};
+    if (!Options_Action_Enum_Parse(*exp_action_str, &exp_action) ||
+        !std::count(valid_types.begin(), valid_types.end(), exp_action)) {
+      throw valhalla_exception_t(144, *exp_action_str);
+    }
+  }
+  options.set_expansion_action(exp_action);
```

Comment:
Sets default to `isochrone` if `action` was omitted from `/expansion` request. Rather throw in that case?

Reply:
yeah i think it should be explicit especially if the output will be potentially different depending on the action. the client should know what they are asking for at some point